### PR TITLE
Add Open procurement endpoints + docker-compose.yaml

### PR DESCRIPTION
**Pull request by OriHoch**
* Added Open Procurement endpoints to /server-urls
* Added docker-compose.yaml which mirrors the docker cloud deployments - added the open procurement container configuration there

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+  kibana:
+    image: kibana:5.3
+
+  govbuy-theme:
+      image: orihoch/open-procure
+      volumes:
+      - ./govbuy-theme:/app/dist/assets
+
+  budgetkey-app-generic-item-govbuy:
```
Comment (akariv) [inline docker-compose.yml:95]: I don't follow... we should use the same service here, not a separate one.

File: server-rules.conf [hunk 2]
```diff
@@ -18,6 +18,14 @@ location /i/ {
     proxy_pass http://budgetkey-app-generic-item:8000;
 }
 
+location /govbuy/search/ {
```
Comment (akariv) [inline server-rules.conf:21]: See comment above

File: docker-compose.yml (hunk 1 above)
Comment (OriHoch) [reply docker-compose.yml:95]: that's what I originally thought but after all the discussions I thought we wanted more separation between the services

practically - if I use the volume mount hack - how would we use the same service?

**Comment by akariv** [comment]
you can map theme.js to theme.govbuy.js.

the server will load all files named theme.*.js

when it receives the 'theme=xxx' query parameter, it will choose the
correct theme and inject it to the client.

**Pull request closed without merging**